clients/hadoopfs/src/test/java/io/lakefs/LakeFSFileSystemTest.java

```diff
@@ -141,6 +151,27 @@ public void setUp() throws Exception {
         fs.initializeWithClient(new URI("lakefs://repo/main/file.txt"), conf, lfsClient);
     }
 
+    /**
+     * @return all pathnames under s3Prefix that start with prefix.  (Obvious not scalable!)
+     */
+    protected List<String> getS3FilesByPrefix(String prefix) throws IOException {
+        final int maxKeys = 1500;
+
+        ListObjectsRequest req = new ListObjectsRequest().
+            withBucketName(s3Bucket).
+            withPrefix(prefix)
+            .withMaxKeys(maxKeys);
```

Comment:
java
```suggestion
        ListObjectsRequest req = new ListObjectsRequest()
                .withBucketName(s3Bucket)
                .withPrefix(prefix)
                .withMaxKeys(maxKeys);
```

Reply:
Done.  The [Google Java Style Guide](https://google.github.io/styleguide/javaguide.html#s4.5.1-line-wrapping-where-to-break) even says explicitly to do it like this even though "_this is not the same practice used in Google style for other languages, such as C++ and JavaScript_"...

Emacs-Lisp command for further reference "if" it happened elsewhere:
```emacs-lisp
(query-replace-regexp "\\.
\\( *\\)" "
\\1." nil nil nil nil nil)
```